roles/ceph-osd/defaults/main.yml

```diff
@@ -2,7 +2,6 @@
 # You can override default vars defined in defaults/main.yml here,
 # but I would advice to use host or group vars instead
 
-raw_journal_devices: "{{ dedicated_devices }}" # backward compatibility with stable-2.2, will disappear in stable 3.1
```

Comment:
It would be better not deleting the variable as it still could be used. Keeping it is a good helper for new comers.